### PR TITLE
Common redistribution clean

### PR DESCRIPTION
**Pull request by clstmbrly**
@lachellel This P.R. includes technical and editorial changes to the FPKI Guides' Trust Stores files (*COMMON removal*).  Chiefly:

- Microsoft_trust_store_verification.md - Added new procedures for verifying the redistribution of COMMON to Windows workstations and devices using BigFix and LANDesk 2016.
- Apple files - Refreshed outdated info. to reflect that Apple *has removed COMMON* from its trust stores and updated related verbiage.
- Made editorial updates (conciseness, consistency, etc.) to the */_includes/* files:  Microsoft and Apple installation, verification, and FAQs, as well as to the */_truststores/* files. 

Please let us know if you have additions or feedback.  Thank you and have a great evening:-).
@ryancdickson

### CODE REVIEW

File: _truststores/00_index.md [hunk 1]
```diff
@@ -5,40 +5,39 @@ collection: truststores
...
 
-Trust Store|Includes FCPCA?<br>(COMMON)|Trust Store Manager|Platforms Serviced|Program Information Location
+Trust Store|Includes FCPCA<br>(COMMON)?|Trust Store Manager|Platforms Serviced|Program Information Location
 ---|---|---|---|---
-Microsoft Trusted Root Certificate Program|Yes (pending removal)|Microsoft Management Console|Windows OS, Internet Explorer Browser, Outlook|http://aka.ms/RootCert
-Apple Root Certificate Program|Yes (pending removal)|Keychain Access Utility|macOS, iOS, tvOS, WatchOS, Safari Browser|https://www.apple.com/certificateauthority/ca_program.html
+Microsoft Trusted Root Certificate Program|Yes (COMMON removal pending)|Microsoft Management Console|Windows OS, Internet Explorer Browser, Outlook|http://aka.ms/RootCert
+Apple Root Certificate Program|No (COMMON previously included in macOS 10.13/iOS 11 and prior versions)|Keychain Access Utility|macOS, iOS, tvOS, WatchOS, Safari Browser|https://www.apple.com/certificateauthority/ca_program.html
```
Comment (lachellel) [inline _truststores/00_index.md]:
```suggestion
Apple Root Certificate Program|No|Keychain Access Utility|macOS, iOS, tvOS, WatchOS, Safari Browser|https://www.apple.com/certificateauthority/ca_program.html
```

Less is more.

File: _truststores/00_index.md [hunk 2]
```diff
@@ -5,40 +5,39 @@ collection: truststores
...
 Mozilla Network Security Services (NSS)|No |Browser trust store|Firefox, Thunderbird, Linux Operating Systems|https://www.mozilla.org/en-US/about/governance/policies/security-group/certs/policy/
 Adobe Approved Trust List|Yes|Application trust store|Adobe Acrobat|https://helpx.adobe.com/acrobat/kb/approved-trust-list2.html
 Java Root Certificate Program|No|Java Applet|Java Distributions|http://www.oracle.com/technetwork/java/javase/javasecarootcertsprogram-1876540.html
 Google|No|Google Admin Console|Android OS, Chromium OS|https://www.chromium.org/Home/chromium-security/root-ca-policy
-Opera|No longer operates its own program and relies on Mozilla
+Opera|No longer operates its own trust store program but relies on Mozilla
```
Comment (lachellel) [inline _truststores/00_index.md]:
```suggestion
Opera|Uses Mozilla
```

File: _truststores/00_index.md [hunk 3]
```diff
@@ -56,19 +55,20 @@ All registry settings for managed Federal Government computers should use group
 
 Federal PKI certificates may be used for digitally signing documents between federal agencies and with business partners.  Adobe is just one option used for digital signatures.
 
-A common question is "which certificate policies are trusted?" You can view the certificate policies that are trusted in Adobe Acrobat:
+A common question is *which certificate policy object identifiers (OIDs) are trusted?* To see the certificate policy OIDs trusted by Adobe Acrobat:
```
Comment (lachellel) [inline _truststores/00_index.md:58]: Revert to previous please.

File: _truststores/02_appl_removal_solutions.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ title: Apple Removal Solutions
 collection: truststores
 permalink: truststores/apple/
 ---
-In September 2018, the U.S. Government is removing the Federal Common Policy Certification Authority (CA) root certificate from the Apple certificate stores. This change will affect all federal agency devices using Apple macOS Mojave, iOS 12, or tvOS 12, and may have an impact on the following services:  
+In September 2018, the U.S. Government removed the Federal Common Policy Certification Authority (CA) root certificate from the Apple certificate stores (also called *trust stores*). This change affects all federal agency devices using Apple macOS Mojave, iOS 12, or tvOS 12 and may have an impact on the following services:
```
Comment (lachellel) [inline _truststores/02_appl_removal_solutions.md]:
```suggestion
In September 2018, the U.S. Government removed the Federal Common Policy Certification Authority (CA) root certificate from the Apple certificate stores. This change affected all federal agency devices using Apple macOS Mojave, iOS 12, or tvOS 12 and may have an impact on the following services:  
```

**Comment by lachellel** [comment]
@clstmbrly minor comments / edits

**Comment by clstmbrly** [comment]
@lachellel All suggested changes have been accepted (new GitHub Beta feature).  Thanks much.

**Pull request merged**